templates/role-assignments/role-assignment-storage-account.json

```
@@ -0,0 +1,43 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "principalId": {
+            "type": "string"
+        },
+        "assignmentType": {
+            "type": "string",
+            "allowedValues": [
+                "StorageBlobDataContributor",
+                "StorageBlobDataReader",
+                "StorageTableDataContributor",
+                "StorageTableDataReader",
+                "StorageQueueDataMessageProcessor",
+                "StorageQueueDataMessageSender"
```

```suggestion
                "StorageQueueDataMessageSender",
                "StorageQueueDataMessageProcessor"
```
so order is consistent with above two sets, write and then read